llama_index/vector_stores/types.py

```diff
@@ -56,22 +57,40 @@ class VectorStoreQueryMode(str, Enum):
     LINEAR_REGRESSION = "linear_regression"
 
 
+class ExactMatchFilter(BaseModel):
+    """Exact match metadata filter for vector stores."""
+
+    key: str
+    value: Union[str, int, float]
+
+
+class MetadataFilters(BaseModel):
+    """Metadata filters for vector stores.
+
+    Currently only supports exact match filters.
+    TODO: support more advanced expressions.
+    """
+
+    filters: List[ExactMatchFilter]
```

Comment:
forgot if i commented this already but we don't support AND and OR yet right?

Reply:
correct, right now we assume AND across all the specified exact match filters.

Reply:
It's a bit trickier to define a general grammar that works with different vector stores.